Server/db/MongoDB.js

```diff
@@ -185,6 +185,21 @@ const requestInviteToken = async (req, res) => {
   }
 };
 
+const getInviteToken = async (req, res) => {
+  try {
+    console.log(req.body.token);
+    const invite = await InviteToken.findOneAndDelete({
+      token: req.body.token,
+    });
+    if (invite === null) {
```

Comment:
maybe 
```javascript 
if (!invite)
```

Reply:
I always prefer strict evaluation whenver possible.  

`!invite` allows for all sorts of things like `0`, `""`, 'undefined`, `NaN` etc that I would definitely _not_ want passing that check :P

Reply:
Good point!